netbox/tenancy/forms/bulk_edit.py

```diff
@@ -127,9 +132,11 @@ class ContactBulkEditForm(NetBoxModelBulkEditForm):
 
     model = Contact
     fieldsets = (
-        FieldSet('group', 'title', 'phone', 'email', 'address', 'link', 'description'),
+        FieldSet('add_groups', 'remove_groups', 'title', 'phone', 'email', 'address', 'link', 'description'),
```

Comment:
Want to move the add/remove fields under a separate "Groups" heading in the form?